repos/system_upgrade/el7toel8/actors/cupsfilterscheck/actor.py

```diff
@@ -0,0 +1,49 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, InstalledRedHatSignedRPM, CupsFiltersModel
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+from subprocess import Popen, PIPE
```

Comment:
I have the feeling that this import is not being used any more. Just for information, Leapp stdlib provides a run() that can be used to execute commands from the system., so subprocess is usually not necessary.